docs/application/dotnet/guides/nui/nui-components/CheckBox.md

```diff
@@ -0,0 +1,78 @@
+# CheckBox
+CheckBox is a common component and describes what action will occur when you select it.  
+A CheckBox can only contain an icon, and can be created using style.
+
+![CheckBox1](./media/CheckBox1.png)
+
+## Create with Style
+
+To create a checkbox using style, follow these steps:
+
+1. Create CheckBox using the default constructor:
+
+    ```cs
+    CheckBox utilityCheckBox = new CheckBox();
+    ```
+
+2. Apply style to the CheckBox:
+
+    ```cs
+    ButtonStyle utilityStyle = new ButtonStyle
+    {
+        Icon = new ImageViewStyle
+        {
+            Size =  new Size(48, 48),
+            ResourceUrl = new Selector<string>
+            {
+                Normal = "btn_check_off.png",
+                Selected = "btn_check_on_24c447.png",
+                Disabled = "btn_check_off.png",
+                DisabledSelected = "btn_check_on_24c447.png",
+            }
+        }
+    }
+    utilityCheckBox.ApplyStyle(utilityStyle);
+    utilityCheckBox.Size = new Size(48, 48);
+    root.Add(utilityCheckBox);
+    ```
+
+Following output is generated when a checkbox is created by using style:
```

Comment:
@surbhig12 Minor question. Don't we need to remove "by" after "using"?

Following output is generated when a checkbox is created by using style:

Reply:
@mijin85cho, I agree. I left this comment thinking it is a minor comment. However, it will be good to fix it.
@zhouleonlei, request you to please incorporate thsi comment as well.

Following output is generated when a checkbox is created using style:

Reply:
Fixed